.releaserc

```diff
@@ -25,24 +25,17 @@
         "changelogFile": "CHANGELOG.md"
       }
     ],
+    [
+      "@semantic-release/npm",
+      {
+        "npmPublish": false
+      }
+    ],
     [
       "@semantic-release/github"
     ],
     [
-      "@semantic-release/git",
```

Comment:
The default option is informed by the plugins that come before this one. Because we used the changelog plugin, this plugin knows that it will commit the asset,  `./CHANGELOG.md`. Because we used the npm plugin, this plugin knows that it will also commit the asset, `package.json`.